index.js

```diff
@@ -0,0 +1,32 @@
+const blocks = document.querySelectorAll(".block");
+blocks.forEach((block) =>  {
+    block.addEventListener("mousedown", () => {
+        block.style.transform = "translate(70px)"
+    });
+    block.addEventListener("mouseup", () => {
+        block.style.transform = "translate(0px)"
+    });
+    block.addEventListener("click", () => {
+        const parent = block.parentNode;
```

Comment:
You don't actually need to set `block.parentNode` to a variable. You could do `block.parentNode.prepend()`

Also, `removeChild()` is unneeded.